crates/tests/signer-tests/verify_proofs.rs

```diff
@@ -0,0 +1,218 @@
+use anyhow::{Ok, Result};
+use node::{
+    BlindedMessage, GetKeysRequest, GetKeysetsRequest, Keyset, MintQuoteRequest, MintRequest,
+};
+use nuts::Amount;
+use nuts::dhke::{blind_message, unblind_message};
+use nuts::nut00::secret::Secret;
+use nuts::nut01::PublicKey;
+use signer::{Proof, VerifyProofsRequest, VerifyProofsResponse};
+use signer_tests::{init_node_client, init_signer_client};
+use starknet_types::Unit;
+
+async fn create_valid_proof(amount: Amount) -> Result<(Proof, Keyset)> {
+    let mut node_client = init_node_client().await?;
```

Comment:
you should not need the node client here.
You can create the keysets from the signer directly
```
message DeclareKeysetRequest {
  string unit = 1;
  uint32 index = 2;
  uint32 max_order = 3;
}
```
you can also get signed messages out of it
```
message SignBlindedMessagesRequest {
  repeated bdhke.BlindedMessage messages = 1;
}
```

Reply:
Basically, remove the dependency to the node you added in this PR, and do it manually, just interacting with the `signer`